modules/widgets/follow-button.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+add_action( 'widgets_init', 'follow_button_register_widget' );
+function follow_button_register_widget() {
+	register_widget( 'Follow_Button_Widget' );
+}
+
+class Follow_Button_Widget extends WP_Widget {
+
+	public function __construct() {
+		parent::__construct(
+			'follow_button_widget',
+			__( 'Follow Button', 'jetpack' ),
+			array( 'description' => __('Add a WordPress.com follow button to allow people to follow your blog easier', 'jetpack' ) )
+		);
```

Comment:
We've tried to add Selective Refresh support to the new Widgets we've added to Jetpack in the past few months. I think it might be nice to add support to that widget too.